### DISCUSSION TITLE
32 bit PC's

### DISCUSSION THREAD

**Discussion by OddCube** [OpenTabletDriver/OpenTabletDriver#2319] [Ideas]
Hey, so im having a 32 bit (windows 7) pc now Im going to buy a wacom so if you could kindly release a version regarding 32 bit os that'd be great :D

If you do, then thank you so much @opentabletdriver team :)

(Optional)
And please don't ask why I am still using 32 bit and windows 7 xD

**Comment by InfinityGhost** [comment]
This is being addressed in #2270, if you'd like you can download a development build [here](https://github.com/OpenTabletDriver/OpenTabletDriver/actions/runs/2503635094) for x86 from this branch.

**Comment by gonX** [comment]
While we don't officially support 32-bit, we have 32-bit releases (called x86) coming with #2270

**Comment by OddCube** [comment]
Thanks InfinityGhost and gonX
